src/sections/Footer.tsx

```diff
@@ -78,19 +83,27 @@ export const Footer = () => {
               ),
             },
             {
-              title: "Über uns",
-              detail: <a className={styles.link}>Unser Zeug</a>,
+              title: t("footer.links.about.header"),
+              detail: (
+                <a className={styles.link}>
+                  {t("footer.links.about.ourStuff")}
+                </a>
+              ),
             },
           ]}
         />
         <div className={styles.legalTexts}>
           <Link to="impressum" className={styles.link}>
-            Impressum
+            {t("footer.imprint")}
           </Link>
           <p>{" & "}</p>
           <Link to="datenschutz" className={styles.link}>
-            Datenschutz
+            {t("footer.privacy")}
           </Link>
+          <p>{" & "}</p>
+          <p className={styles.languageSwitch} onClick={handleLanguageChange}>
+            {i18n.language == "de-DE" ? "DE" : "EN"}
+          </p>
```

Comment:
Elements that can be clicked should always be an `<a>`, `<button>`, `<input>`, etc. or have the ARIA attribute [`role="button"`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role).

Regarding UX/accessibility, I think it would be better if there was a button for each language (<kbd>Deutsch</kbd> <kbd>English</kbd>) to make it easier to use. Currently it's unclear what clicking on `DE` does.

Reply:
It is just the solution for now. later we will do a solution with better UX

Reply:
Yeah the button the button is only temporarily there. I will create an issue for that.

Reply:
That's ok. But please make sure we have an issue to track this when this PR is merged.

Reply:
#74